app/views/user_mailer/welcome.html.erb

```diff
@@ -1,27 +1,26 @@
 <p>
-  Welcome to Operation Code! You've just joined a community of 3000+ coders, who are all looking to further their skills and careers in software development!<br />
+  Welcome to Operation Code!<br />
 </p>
 
+<p>
+  Be on the lookout for an email invite to the <a href="https://operation-code.slack.com">Operation Code Slack community</a>. Slack is where we communicate online in real-time to ask questions, collaborate, and share information. It's a pivotal aspect of our community! If you've never used Slack before, consider watching this <a href="https://www.youtube.com/watch?v=9RJZMSsH7-g">instructional video</a> on using the application. <br />
+</p>
 
 <p>
-  We are a decentralized organization, and we work both <a href="https://community.operationcode.org/">online</a> and in <a href="https://www.meetup.com/pro/operationcode/">local chapters</a> across the country. Please introduce yourself on our <a href="https://community.operationcode.org/c/introductions">forums</a>, and come join one of our <a href="https://www.meetup.com/pro/operationcode/">meetups in your local area!</a> <br />
-  Also, be sure to check your email for an invite to the Operation Code Slack community (<a href="https://operation-code.slack.com">https://operation-code.slack.com</a>).<br />
-  Slack is where we communicate online in real-time to ask questions, collaborate, and share information. Once in Slack (you'll probably want to download the app), introduce yourself in the #general channel. An example could be: <br />
+  You've just joined a community of 3000+ software developers, veterans, and military spouses, who are all looking to further their skills and careers in tech! We are a decentralized organization, and we work both online (via Slack) and in <a href="https://www.meetup.com/pro/operationcode/">local chapters across the country.</a> We highly recommend giving yourself an introduction in the #general channel on Slack, and to sign up to go to your next local meetup! An example could be: <br />
   "Hi all. My name is David. I'm from Washington and I'm interested in web development!"
   Please read our <a href="http://op.co.de/code-of-conduct">Code of Conduct</a>, as well.
```

Comment:
Is this the right code of conduct?
https://github.com/OperationCode/operationcode_docs/blob/master/community/code_of_conduct.md